### PR TITLE
azcopy: add page

### PR DESCRIPTION
**Pull request by skitau**
Next page covering my common tools for my Hacktoberfest challenge to commit to public repos.

Hopefully I haven't missed any formatting items in this one. :)

Happy Hacktoberfest!

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]

File: pages/windows/azcopy.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Login to an Azure Tenant:
+
+`azopy login`
+
+- Upload a local file:
+
+`azcopy copy '{{path/to/source/file}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}/{{blob-name}}'`
+
+- Upload all .txt and .jpg files:
```
Comment (navarroaxel) [inline pages/windows/azcopy.md:14]: Is this better? I think the extensions should be with back tick.
```suggestion
- Upload files with `.txt` and `.jpg` extensions:
```

**Review by navarroaxel — COMMENTED** [review]
Thanks @skitau for your submission, I left a comment.

**Comment by skitau** [comment]
> Thanks @skitau for your submission, I left a comment.

Agree, have included.

**Review by navarroaxel — APPROVED** [review]

File: pages/windows/azcopy.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`azopy login`
+
+- Upload a local file:
+
+`azcopy copy '{{path/to/source/file}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}/{{blob-name}}'`
+
+- Upload files with `.txt` and `.jpg` extensions:
+
+`azcopy copy '{{path/to/source}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}' --include-pattern '*.txt;*.jpg'`
```
Comment (sbrl) [inline pages/windows/azcopy.md:16]: Missing token syntax here?
```suggestion
`azcopy copy '{{path/to/source}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}' --include-pattern '{{*.txt;*.jpg}}'`
```

Comment (skitau) [reply pages/windows/azcopy.md:16]: Ah good catch! Addressed with #4674

**Pull request merged**